### PR TITLE
need models to store commenting settings

### PR DESCRIPTION
**Pull request by Rizziepit**

### CODE REVIEW

**Comment by Rizziepit** [comment]
@miltontony @smn ready for review. You can ignore `ViewTestCase` - I just moved it.

Any comment stream can now have metadata. By default any stream has empty metadata (`{}`) associated with it. Example of metadata:

``` python
metadata = {
    'state': 'closed',
    'X-foo': 'bar'
}
```

`state` is a known key - posting a comment will return 403 if it's `closed` or `disabled`. Any key starting with `X-` is arbitrary data that the client determines.

Conceptually a stream always has metadata whether there is a DB record or not. This makes it easier for the client to keep stream metadata in sync with the related content since it doesn't have to do "if exists put else post". This also simplifies updating multiple streams' metadata in one go.

File: unicore/comments/service/models.py [hunk 1]
```diff
@@ -16,11 +16,14 @@
     (u'removed_by_moderator', u'Removed by moderator'),
     (u'removed_by_community', u'Removed by community'),
     (u'removed_for_profanity', u'Removed for profanity'))
+COMMENT_STREAM_STATES = (u'open', u'closed', u'disabled')
```
Comment (smn) [inline unicore/comments/service/models.py:19]: Tiny preference for making these upper case since it's mostly intended to be treated as a constant?

**Comment by smn** [comment]
:+1: from me, looking good. @miltontony ?

**Comment by miltontony** [comment]
:+1:

**Pull request merged**